packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -127,6 +128,7 @@ function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLEl
         role="group"
         className={classNames(styles, 'spectrum-SearchWithin', styleProps.className)}
         ref={groupRef}>
+        <VisuallyHidden id={visuallyHiddenId} data-testid="searchwith-label">{formatMessage('searchWithin')}</VisuallyHidden>
```

Comment:
btw, I don't think `data-testid` is removed anymore in published builds. maybe we can avoid it and change the test instead?

Reply:
Wait, when did that stop?

Reply:
When we stopped using babel